commands/project/tags/delete/mod.ts

```diff
@@ -0,0 +1,102 @@
+import { projects, projectTags } from "../../../../api/projects.ts";
+import { asserts } from "../../../../lib/asserts.ts";
+import { args, command, flag, flags, z } from "../../../../zcli.ts";
+import { select } from "../../../../prompts/select.ts";
+import { dataTable } from "../../../../lib/data-table.ts";
+import { pickJson } from "../../../../lib/pick-json.ts";
+import { loading } from "../../../../lib/loading.ts";
+import { defaultFields } from "../mod.ts";
+
+/**
+ * This variable is automatically generated by `zcli add`. Do not remove this
+ * or change its name unless you're no longer using `zcli add`.
+ */
+const subCommands: ReturnType<typeof command>[] = [];
+
+export const delete_ = command("delete", {
+  short: "Delete a project tag.",
+  long: `
+    Delete a project by its name. If you don't provide a name, this command will
+    prompt you for one.
```

Comment:
Looks like youre prompting both, projects and tags if they dont exists. Wonder if the long description should say both, instead of just name. Also, is name what we are using across the app? I feel like name is confusing since it is an id (team id) what it needs.